internal/client/client.go

```diff
@@ -2,38 +2,27 @@ package client
 
 import (
 	"context"
-	"crypto/tls"
-	"crypto/x509"
 	"fmt"
-	"io/ioutil"
-	"net/http"
-	"os"
-	"path/filepath"
 
 	"github.com/hashicorp/secrets-store-csi-driver-provider-vault/internal/config"
 	"github.com/hashicorp/vault/api"
-	"github.com/pkg/errors"
-	"golang.org/x/net/http2"
 )
 
-func New(params config.Parameters) (*api.Client, error) {
+func New(vaultAddress string, tlsConfig config.TLSConfig) (*api.Client, error) {
```

Comment:
Do we have documented somewhere the names we expect these files to have? In Helm we can setup things correctly but if a user tries to install this manually, it might be a gotcha.

Reply:
The secret provider class config takes in file names, so I think they can be anything unless I'm missing something else, e.g. `vaultTLSClientCertPath: /mnt/tls/client.crt`.

Reply:
Ah OK, I assumed `path` was just the path, like we do in Vault's configuration.